code/game/turfs/turf.dm

```diff
@@ -241,12 +240,62 @@
 
 		C << "<span class='notice'>You slipped[ O ? " on the [O.name]" : ""]!</span>"
 
+		if(C.dna)
+			if(C.dna.features["tail_human"] == "Cat" && C.dna.features["ears"] == "Cat")
+				C.SpinAnimation(7,1)
+				visible_message("<span class='notice'>[C.name] slips... but manages to stick the landing with their cat like reflexs!</span>")
+				return 0
+		var/list/impact_point = pick("Head", "Chest", "Back", "Ass", "Hands", "Groin")
+		var/message
+		switch(impact_point)
+			if("Head") //oh boy
+				if(istype(C.head,/obj/item/clothing/head))
+					var/obj/item/clothing/head/H = C.head
+					if(H.armor["melee"])
+						playsound(C, 'sound/effects/woodhit.ogg', 30, 1)
+						message = "[C.name] slips backwards straight onto their head! You hear a crack as their [H.name] takes the impact!"
+						H.armor["melee"] = 0
+						H.desc += " A nasty looking crack runs right up the middle, this won't be good for stopping impacts anymore."
+				if(!message) //Should have worn your helmet!
+					if(prob(25))
+						playsound(C, 'sound/effects/blobattack.ogg', 30, 1)
+						message = "[C.name] slips backwards straight onto their head! You hear a sickening crack as their skull splits open!"
+						C.apply_damage(75, BRUTE, "head")
+						C.setBrainLoss(50)
+					else
+						playsound(C, 'sound/effects/bang.ogg', 30, 1)
+						message = "[C.name] slips backwards straight onto their head! You hear an unnerving whump as their head bounces off the floor!"
+						C.apply_damage(20, BRUTE, "head")
+						C.setBrainLoss(10)
+			if("Chest")
+				message = "[C.name] slips fowards flat onto their chest, knocking the air out of their lungs!"
+				C.setOxyLoss(50)
+
+			if("Back")
+				message = "[C.name] slips backwards flat on their back!"
+				C.apply_damage(10, BRUTE, "chest")
+
+			if("Ass") //Tactically speaking, this is the best place to land
+				message = "[C.name] slips, landing squarely on their ass!"
+				C.apply_damage(5, BRUTE, "chest") //Where's my ass damage zone?
```

Comment:
Apply 5 damage to both of their legs instead, falling on your ass should hurt like a bitch.